dbms/src/Flash/Planner/PhysicalPlanNode.h

```diff
@@ -56,6 +60,10 @@ class PhysicalPlanNode
 
     virtual void transform(DAGPipeline & pipeline, Context & context, size_t max_streams);
```

Comment:
As I mentioned in my previous comment, this `transform` name conflicts which the other usage and is misleading. Maybe we can rename it to `buildBlockInputStream`?

This way, our system has `BlockInputStream`-based pull-model.

Reply:
ok, updated.